### DISCUSSION TITLE
这是群组关闭了，还是我被拉黑了？

### DISCUSSION THREAD

**Discussion by Rangtian** [mack-a/v2ray-agent#783] [General]
![image](https://github.com/mack-a/v2ray-agent/assets/22640407/3a18b150-5eb8-4bb7-b100-7dca7057bfe9)

技术小木屋 channel 的讨论为啥也看不到？总不能是被拉黑了吧？我明明从未加入过讨论群啊？

![image](https://github.com/mack-a/v2ray-agent/assets/22640407/e3ee0f60-c351-4599-a951-8c622a636a28)
![image](https://github.com/mack-a/v2ray-agent/assets/22640407/c1c70d0b-2dbe-4981-a3b8-c0d965706f2f)

**Comment by mack-a** [comment]
不知道呢 可以去机器人申请解锁

**Reply by Rangtian** [reply]
请问怎么申请？我点「JOIN GROUP」就提示：Sorry, this group is not accessable.

我ID是 CC 8.79，如果被群组拉黑了，还望解封，谢谢你。

**Reply by mack-a** [reply]
https://t.me/technologyshare_bot 机器人申请

**Reply by Rangtian** [reply]
谢谢你，已申请。

**Comment by mack-a** [comment]
你的账号被机器人识别为广告被封了 可以修改下昵称然后重新申请

**Reply by Rangtian** [reply]
已改，谢谢你
